### ISSUE TITLE
Add bananos support

### ISSUE THREAD

**Issue by ghost** [rotilho/jnano-client#19]
Could you add bananos support to jnano?

It's as simple as adding ban_ to the places where you support xrb_ and nano_

I've found it in these places so far:
NanoBlocks.hashStateBlock

NanoAccounts.ACCOUNT_REGEX

**Comment by rotilho** [comment]
Sure! I'll work on that tomorrow :)

**Comment by rotilho** [comment]
Hello @coranos I created two PR to add support Banano. I decided to provide a future proof implementation which will support multiple account types. This solutions seems more robust and it won't cause possible client errors (i.e. someone using a banano address in nano application).

Let me know your thoughts. I'm planning to release tomorrow before my holidays

https://github.com/rotilho/jnano-client/pull/20
https://github.com/rotilho/jnano-commons/pull/2

**Comment by ghost** [comment]
sounds good to me. will pull and test when I get some time

**Comment by rotilho** [comment]
@coranos due to time constraints I'll be releasing a new version in few minutes

**Comment by rotilho** [comment]
New version released https://github.com/rotilho/jnano-client/releases/tag/1.1.0

**Comment by ghost** [comment]
Found a bug, when I call transactionOperations.send it converts the banano address to nano.

The reason is here:
https://github.com/rotilho/jnano-client/blob/master/src/main/java/com/rotilho/jnano/client/transaction/NanoTransactionOperations.java#L155

NanoAccounts.createAccount(publicKey) 

```
    public static String createAccount(@NonNull byte[] publicKey) {
        return createAccount(NanoBaseAccountType.NANO, publicKey);
    }
```

**Comment by rotilho** [comment]
Hello Caroas,

Thanks for the feedback. I made some changes in the tests to use a custom prefix so all issues regarding account type should be addressed in the just released version `1.1.1`.
